starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -564,7 +564,7 @@ tasks.withType(JavaCompile) {
     ])
 }
 }
-@if (features.language().isKotlin() || features.testFramework().isKotlinTest()) {
+@if (features.language().isKotlin() || features.testFramework().isKotlinTest() || features.testFramework().isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411